contracts/ISSVNetworkCore.sol

```diff
@@ -7,7 +7,7 @@ interface ISSVNetworkCore {
     /***********/
 
     struct Validator {
-        address owner;
+        bytes32 hashedOperatorIds;
```

Comment:
what is gas difference in hashing operator id vs array for 4 and 13

Reply:
4 operators:
hashed operator ids: 245600
raw operator ids: 268791

13 operators:
hashed operator ids: 459600
raw operator ids: 552531

Reply:
@lior-blox Way too expensive, lets keep as is